apollo-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'kotlin'
+apply plugin: 'idea'
```

Comment:
This plugin adds tasks for generating the `.idea` files needed by Android Studio/IntelliJ. It has nothing to do with UI AFAIK.
I personally haven't used it much since these files are generated when you open the project in the IDEA, but they seem pretty standard in most java projects.
If you don't like it I can remove it.